tests/pfcwd/conftest.py

```diff
@@ -244,3 +244,28 @@ def pfcwd_pause_service(ptfhost):
         needs_resume["garp_service"] = False
 
     logger.debug("pause_service needs_resume {}".format(needs_resume))
+
+
+@pytest.fixture(scope="function", autouse=False)
+def skip_pfcwd_higher_speeds(
+        duthosts,
+        enum_rand_one_per_hwsku_frontend_hostname,
+        setup_pfc_test):
+    duthost = duthosts[enum_rand_one_per_hwsku_frontend_hostname]
+    if duthost.facts["asic_type"] != "cisco-8000":
```

Comment:
Hi @kevinskwang  , please take a look at this logic.
Dut to the 400G port/fanout can't be stably trigger the pfcwd, we plan to xfail it on Cisco 8000 platforms.
For now the logic covers T0/T1/T2,
Please let me know if you want to keep the old logic of T0/T1 devices, we can update it to cover T2 only

Reply:
how to cover pfcwd function after skip?

Reply:
> how to cover pfcwd function after skip?

@kevinskwang it's not skipping, but xfail.
If it passes it shows xpass, if it fails it shows xfail. Both of then will not be considered as failed.
So it's still got covered but won't affect passing rate.

And in chassis, we will cover this scenario on Ixia RDMA tests. So want you know whether you prefer to adopt the xfail logic into t0/t1 also, or keep current but still flakness

Reply:
let's  keep it as it is in pizzabox. I don't think we have enough relacement at current stage.

Reply:
@rraghav-cisco please constraint the xfail logic on Cisco chassis only, thanks!
@abdosi for viz

Reply:
@yejianquan : Its already xfail only for cisco-8000.

Reply:
no, you need to just skip for 8800